tests/sparseml/onnx/optim/test_analyzer_model.py

```diff
@@ -162,14 +167,14 @@ def test_model_analyzer(analyzer_models):  # noqa: F811
     expected_outputs = [x for x in expected_outputs if x]
     result = [False] * len(expected_outputs)
     for i, expected_output in enumerate(expected_outputs):
-        result[i] = _test_model_analyzer(model_path, expected_output)
+        _test_model_analyzer(model_path, expected_output)
     """
     If we have only one test case, it must must evaluate to True,
     If we have two test cases, at least one must evaluate to True.
     In other words, we are happy with test passing for legacy or
     upgraded PyTorch (worst case scenario).
     """
-    assert any(result)
+    # assert any(result)
```

Comment:
My conviction is that we should not have commented out code in the production code. If its commented out, it should probably get deleted.